src/ILP/model/account-balance.ts

```diff
@@ -0,0 +1,92 @@
+import bigInt, { BigInteger } from 'big-integer'
+import { GetBalanceResponse } from '../../generated/node/ilp/get_balance_response_pb'
+
+/**
+ * Response object for requests to get an account's balance
+ */
+export class AccountBalance {
+  /**
+   * The accountId for this account balance.
+   */
+  readonly accountId: string
```

Comment:
Please use `ID` instead of `Id` (here + comments)

Reply:
I'm OK with this, but the Hermes protobufs use `accountId`, so there will be some inconsistency there.  We also use `accountId` across the whole java ILP stack, so it will be inconsistent with that.

Reply:
I'm against doing this as it represents a relatively drastic refactoring to a number of applications and libraries in our domain.

Reply:
Is there a desire to follow naming conventions of the language, or use the same naming standard across all languages. In Javascript, `accountId` is more common. For example it's `document.getElementById()` not `document.getElementByID()`